features/google/src/model/importers/activities-importer-json.js

```diff
@@ -27,12 +37,14 @@ export default class ActivitiesJsonImporter {
             matchRegex(path, this)
         );
         const parser = new ActivityJsonParser();
+        const parserOutput = await Promise.all(
+            activityEntries.map((entry) => parser.parse(entry))
+        );
         googleAccount.activities.push(
-            ...(
-                await Promise.all(
-                    activityEntries.map((entry) => parser.parse(entry))
-                )
-            ).flat()
+            ...parserOutput.map((output) => output.userActivity).flat()
```

Comment:
Same code as https://github.com/polypoly-eu/polyPod/pull/981/files#diff-06bfcde4b6890e277c71320bbe9ec82a280c8f879e4134aa030456c6f1b43304R63
why not spin it off to a base class?

Reply:
Done.